endpoints/cookie_sync.go

```diff
@@ -427,6 +427,24 @@ func (c *cookieSyncEndpoint) handleResponse(w http.ResponseWriter, tf usersync.S
 		})
 	}
 
+	if debug {
+		biddersSeen := make(map[string]struct{})
+		var debugInfo []cookieSyncResponseDebug
+		for _, bidderEval := range biddersEvaluated {
+			var debug cookieSyncResponseDebug
```

Comment:
Nitpick: `if debug {` and `var debug cookieSyncResponseDebug` create an ambiguity in variables names. 
Can you rename `var debug cookieSyncResponseDebug` to `var debugResponse cookieSyncResponseDebug`, or something else that makes sense to you?